Sources/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -91,6 +95,19 @@ class AttachmentDTO: NSManagedObject {
     }
 }
 
+extension AttachmentDTO: EphemeralValuesContainer {
+    func resetEphemeralValues() {
+        switch localState {
+        case .downloading, .downloadingFailed:
+            localState = nil
+            localURL = nil
+            localRelativePath = nil
+        default:
+            break
+        }
+    }
+}
```

Comment:
Need to reset the downloading state if the app quits while downloading the file.